src/pinefarm/install.py

```diff
@@ -258,6 +258,8 @@ def lhapdf_conf(pdf):
         LHAPDF name of the required PDF
 
     """
+    if os.environ("CI"):
```

Comment:
The problem is here, but it's a real problem, so we should apply a suitable fix.

`lhapdf-config` has been compiled somewhere else, and this makes its result wrong (we could actually consider fixing in the container generation as well, but it's too much work for now).
However, we are able to specify the correct path with the env var, but it's ignored because `lhapdf-config` is available. It's clear that the env var should have priority.

Thanks for having found the bug, now let's apply the fix.